source/globalCommands.py

```diff
@@ -1956,6 +1961,57 @@ def script_reportOrShowFormattingAtCaret(self, gesture):
 		elif repeats == 1:
 			self.script_showFormattingAtCaret(gesture)
 
+	@script(
+		description=_(
+			# Translators: the description for the reportDetailsSummary script.
+			"Report summary of any annotation details at the current location."
+		)
+	)

```

Comment:
Without assigning a category to this script it would show in the Miscellaneous category in the input gestures dialog. I'm not against that - just wanted to make sure this was though through rather than done on accident.

Reply:
Thanks for the reminder, I hadn't thought too hard about which category it should belong to yet. I think it makes most sense in the system caret category, what do you think?

Reply:
> I hadn't thought too hard about which category it should belong to yet. I think it makes most sense in the system caret category, what do you think?

Agreed.